src/test/data/JsonSerializableAddressBookTest/typicalBuyersAddressBook.json

```diff
@@ -0,0 +1,53 @@
+{
+  "_comment": "AddressBook save file which contains the same Person values as in TypicalBuyers#getTypicalAddressBook()",
+  "buyers" : [ {
+    "personCategory" : "Buyer",
+    "name" : "Alice Pauline",
+    "phone" : "94351253",
+    "email" : "alice@example.com",
+    "address" : "123, Jurong West Ave 6, #08-111",
+    "tagged" : [ "friends" ]
```

Comment:
I guess can remove all the tag instances here, since we do not have any Tag class

Reply:
fixed